modules/SofaGeneralLoader/GeneralLoader.h

```diff
@@ -1,33 +0,0 @@
-/******************************************************************************
```

Comment:
Hello Erik, 

I'm not sure this one should be removed, on the contrary its use should be generalized. Additionally I think it should also define the SOFA_TARGET to SofaGeneralLoader.

Please confirm @guparan

Reply:
Ok I sent a message to @guparan already regarding this.
Right now all macros are defined in sofa-build\include\SofaMisc\config.h etc..
This class is not included. Let me know which one should be kept. I'll revert the commit if needed.

Reply:
I will report our discussion in an issue and remove that change from the PR to have only changes on header includes.